Tests/OpenCombineTests/Helpers/TestHelper.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  TestHelper.swift
+//  
+//
+//  Created by Joseph Spadafora on 7/6/19.
+//
+
+import XCTest
+
+#if OPENCOMBINE_COMPATIBILITY_TEST
+import Combine
+#else
+import OpenCombine
+#endif
+
+@available(macOS 10.15, *)
+class TestHelper<Value: Equatable,
```

Comment:
There is a lot of really similar boilerplate in the tests for setting up the custom subscriptions, custom publisher and tracking subscribers.  This makes setting up a test case as simple as passing a closure to create the chained subscriber.  Let me know what you think @broadwaylamb .  If not exactly this, something along these lines would help remove a lot of the duplicate code in the testing suites.

Reply:
I think it's a great idea!

Reply:
👍